js/tests/unit/modal.spec.js

```diff
@@ -595,12 +595,11 @@ describe('Modal', () => {
       modal.show()
     })
 
-    it('should not close modal when escape key is pressed with keyboard = false and backdrop = static', done => {
-      fixtureEl.innerHTML = '<div class="modal" data-bs-backdrop="static" data-bs-keyboard="false"><div class="modal-dialog"></div>'
+    it('should not close modal when escape key is pressed with keyboard = false', done => {
+      fixtureEl.innerHTML = '<div class="modal"><div class="modal-dialog"></div>'
```

Comment:
Aren't we missing a `</div>` here BTW? It doesn't affect anything, and we don't enforce it, though.

Reply:
Updated @XhmikosR 👍